Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -143,6 +187,37 @@ public bool ExpandedState
             }
         }
 
+        private static readonly string _allResultsFilter = TestExplorer.ResourceManager.GetString("TestExplorer_AllResults", CultureInfo.CurrentUICulture);
+        private string _selectedOutcomeFilter = _allResultsFilter;
+        public string SelectedOutcomeFilter
+        {
+            get => _selectedOutcomeFilter;
+            set
+            {
+                if (_selectedOutcomeFilter != value)
+                {
+                    _selectedOutcomeFilter = value.Replace(" ", string.Empty);
+                    OnPropertyChanged();
+                    Tests.Filter = FilterResults;
+                    OnPropertyChanged(nameof(Tests));
```

Comment:
Same as the `TestNameFilter` setter. I wonder why you're using string based filtering if you do have a `TestFiltering` property that is of type `TestOutcome?`

Reply:
The `ComboBox` items used for the outcome filter are string values because I wasn't able to figure out how to display the images in the dropdown.